.gitignore

```diff
@@ -42,3 +42,4 @@ user.props
 # Generated rst files
 ######################
 genrst/
+
```

Comment:
Can be dropped, my setup generally complains about no newlines at the end of files.
I shall update.